python/tests/avro/test_decoder.py

```diff
@@ -24,8 +24,7 @@
 from unittest.mock import MagicMock, patch
 
 import pytest
-
-from pyiceberg.avro.decoder import BinaryDecoder, StreamingBinaryDecoder, new_decoder
+from pyiceberg.avro.decoder import BinaryDecoder, StreamingBinaryDecoder, new_decoder  # type: ignore
```

Comment:
It was throwing Module "pyiceberg.avro.decoder" has no attribute "new_decoder" while running mypy check in commit hook hence added
# type: ignore comment to tell mypy to ignore this error